src/main/java/com/cgdpd/library/entity/BookCopyEntity.java

```diff
@@ -41,6 +48,16 @@ public class BookCopyEntity {
           name = "user_id",
           referencedColumnName = "id",
           foreignKey = @ForeignKey(name = "fk_book_copies_user_id"))
-    @ManyToOne
+    @ManyToOne(fetch = LAZY)
     private UserEntity userEntity;
+
+    @Override
+    public String toString() {
```

Comment:
Due to the use of lazy fetching for entity relations, the default `toString()` method provided by Lombok's `@Data` annotation can't be used. Using it will result in a `LazyInitializationException` because it tries to call the `toString()` of the related entity. To prevent this, we've manually implemented the `toString()` method, ensuring only the IDs of the related entities are used, e.g., `bookEntity.getId()`.